Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN build_platform() { \
          # Add the specified Rust target architecture
         rustup target add $ARCH; \
         # Update package lists and install the specified compiler
-        apt-get update && apt-get -y install $COMPILER; \
+        apt-get update && apt-get -y install $COMPILER libclang-dev; \
```

Comment:
why do we need this?

Reply:
When I tried to build the image locally, I was getting this error due to the c-kzg patch

```bash
thread 'main' panicked at 'Unable to find libclang: "couldn't shared libraries matching: ['libclang.so',
'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where
one of these files can be found
```